### ISSUE TITLE
Is this repo still maintained?

### ISSUE THREAD

**Issue by jakubzitny** [meltingice/psd.js#93]
Hi @meltingice,

I am wondering what is the status and future of this project from your side.

It seems there is a number of relevant issues and PRs with no maintainer activity. There might be some developers that would help you with it, or you can even transfer the repo to someone else.

Would you be willing to cooperate in any way?

Thanks.

**Comment by fabulousduck** [comment]
i can second what @jakubzitny is saying and would love to help maintain this project as it would be an awful lot of trouble for someone to have to rebuild this.

**Comment by meltingice** [comment]
I'm open to help maintaining the project. Do you have much Javascript experience @fabulousduck and have you messed around with the PSD file format at all? I think some basic maintenance will be sufficient for now until I can rewrite the project in vanilla Javascript to drop the coffeescript dependency.

**Comment by fabulousduck** [comment]
@meltingice i currently work at a company that is fully built on a JS stack and i work with it on a day to day basis. As for working with PSDJS, i have created a full PSD->HTML converter in it for the company i currently work at. But i do need to stress that maintaining this would require me to delve into coffeescript some more to fully understand the project and be able to make informed decisions about it and reason about changes in PR's and such.


As for the complete rewrite of the project in vanillaJS as you mentioned, i would be more than glad to help out with that as i have 60% completed one myself in a c++ wrapper for node using the V8 engine and have the knowledge of the PSD file format to be of use.

Let me know what you think @meltingice.

**Comment by fabulousduck** [comment]
Another suggestion is that PSDJS is moved to a company repo, this would allow for maintainers to easily work on it and review PR's while keeping @meltingice at the helm as it is still his project and work on the vanilla rewrite of the project.

**Comment by meltingice** [comment]
Let's leave it as it is for now. Working with and approving pull requests will go a long way in helping me out. Once they're approved, I can merge. Adding you as a contributor @fabulousduck.

**Comment by meltingice** [comment]
I'm also going to start on the coffee -> JS conversion this week. Will ping you if I need help.

**Comment by fabulousduck** [comment]
@meltingice alright, i will get to reading PR's today and notify you about any PR's that need merging.

**Comment by fabulousduck** [comment]
@meltingice are you ok with me closing some old issues that were either resolved or irrelevant ?

**Comment by meltingice** [comment]
@fabulousduck yeah go for it. I got a good start on the vanilla JS branch last night. Header + layer comp parsing is implemented.

**Comment by fabulousduck** [comment]
@meltingice will do, and i read through a bit of the vanilla code which is looking good, keep it up 👍
